strings/src/main/java/com/revenuecat/purchases/strings/ConfigureStrings.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases.strings
+
+object ConfigureStrings {
+    const val APP_BACKGROUNDED = Emojis.INFO + " App backgrounded"
+    const val APP_FOREGROUNDED = Emojis.INFO + " App foregrounded"
+    const val DEBUG_ENABLE = Emojis.INFO + " Debug logging enabled"
+    const val INVALID_CREDENTIALS_ERROR = Emojis.ROBOT + Emojis.ERROR + " There was a credentials issue. " +
```

Comment:
A couple of thoughts on the emojis variables: 
- I think we should name them after the expected use case, ie: `SAD_CAT_EYES` -> `SERVER_ERROR` or something, `ROBOT` -> `CONFIGURATION_ERROR`. That way we make the intent clear to the programmers, and they don't have to wonder whether a given situation merits a sad cat or a robot, and we also gain the ability to switch out the specific emoji should we ever want to, as long as the intent remains the same. 
- I think it'd be nice if we could tie the emojis to the `log` methods - i.e: `logInfo` uses the `Emojis.INFO`, or it could even be a param. that way we don't have to repeat the mapping all over the strings files, and we're guaranteed that the emojis will be there. Nothing's stopping us from having multiple `log` methods or an extra param in them, and it might help avoid issues.

Reply:
- I like the idea of making the Emoji names more description of the use case. The original idea was to include an emoji legend in the docs near where we tell developers to enable debug logs. Regardless of if we flip debug logs to be on automatically or not, I like the idea of naming them after the use case. 
- I like the idea of creating multiple `log` methods with an extra param in them. I'd rather avoid having an emoji tied to a log level since there could be multiple emojis depending on the message. Due to this I'm thinking the param should take in a list of emojis.

Reply:
https://docs.revenuecat.com/docs/errors#legend 
Going to follow this legend so `SAD_CAT_EYES` -> `RC_ERROR`, `ROBOT` -> `GOOGLE_ERROR`, `APPLE` -> `APPLE_ERROR`.

Reply:
is this going to be done in a different PR?